.travis/build.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
Rather than just remove `echo_run`, I've setup the script to run with `[set -x](https://stackoverflow.com/a/36277661/2471708)` which provides (imo) more robust debugging. You can see the execution line-by-line, step-by-step in the [output](https://travis-ci.org/apmccartney/libraries/jobs/246701777#L849).